admin_guide/scheduling/descheduler.adoc

```diff
@@ -0,0 +1,326 @@
+onto more appropriate nodes. 
+
+Your cluster can benefit from descheduling and rescheduling already-running pods for various reasons:
+
+* A nodes is under- or over-utilized.
+* Pod and node affinity requirements, such as taints or labels, have changed and the original scheduling decisions are longer appropriate for certain nodes.
+* Node failure requires pods to be moved.
+* New nodes are added to clusters.
+
+The descheduler does not schedule replacement of evicted pods. The xref:../../admin_guide/scheduling/scheduler.adoc#admin-guide-scheduler[scheduler]
+automatically performs this task for the evicted pods.
+
+[NOTE]
+====
+The descheduler job is considered a _critical pod_, which prevents the descheduler pod from being evicted by the descheduler. 
+You can make a pod a critical pod by adding the the `scheduler.alpha.kubernetes.io/critical-pod` annotation to the pod specification 
+====
+
+The descheduler job and descheduler pod are created in `kube-system` project.
+
+[IMPORTANT]
+====
+The descheduler is a Technology Preview feature only.
+ifdef::openshift-enterprise[]
+Technology Preview features are not supported with Red Hat production service
+level agreements (SLAs), might not be functionally complete, and Red Hat does
+not recommend to use them for production. These features provide early access to
+upcoming product features, enabling customers to test functionality and provide
+feedback during the development process.
+
+For more information on Red Hat Technology Preview features support scope, see
+https://access.redhat.com/support/offerings/techpreview/.
+endif::[]
+====
+
+The descheduler does not evict the following types of pods:
+
+* Critical pods (with the `scheduler.alpha.kubernetes.io/critical-pod` annotation).
+* Pods (link:https://kubernetes.io/docs/tasks/administer-cluster/static-pod/[static and mirror pods] or pods in standalone mode) not associated with a Replica Set, Replication Controller, Deployment, or Job (because these pods are not recreated).
+* Pods associated with xref:../../dev_guide/daemonsets.adoc#dev-guide-daemonsets[DaemonSets].
+* Pods with local storage.
+* Pods subject to xref:../../admin_guide/managing_pods.adoc#managing-pods-poddisruptionbudget[Pod Disruption Budget (PDB)] 
+are not evicted if descheduling violates the PDB. The pods can be evicted using 
+an xref:../../admin_guide/out_of_resource_handling.adoc#out-of-resource-eviction-policy[eviction policy].
+
+[NOTE]
+====
+Best efforts pods are evicted before Burstable and Guaranteed pods.
+====
+
+To use the descheduler:
+
+. xref:admin-guide-descheduler-role[Create a role].
+. Define the descheduling behavior in a xref:admin-guide-descheduler-policies[policy file]. 
+. Create a xref:admin-guide-descheduler-policy[configuration map to reference the policy file].
+. Create the xref:admin-guide-descheduler-job[descheduler job configuration].
+. xref:admin-guide-descheduler-run[Run the descheduler job].
```

Comment:
I was confused about the above. At first, I thought these were prerequisites, or links to outside content, but I now see we're describing the following sections. Can I suggest rewording the intro sentence?

"The following sections describe the process to configure the descheduler."

I'm not one for the list of sections in the topic (Minimalism! They're listed in other places!), but I know I might be alone in this. A reword of the intro might help my brain.